src/Azure.Functions.Cli/ExtensionBundle/ExtensionBundleConfigurationBuilder.cs

```diff
@@ -26,7 +20,7 @@ public void Configure(IConfigurationBuilder builder)
             {
                 builder.AddInMemoryCollection(new Dictionary<string, string>
                 {
-                    { "AzureFunctionsJobHost:extensionBundle:downloadPath", Path.Combine(Path.GetTempPath(), "Functions", ScriptConstants.ExtensionBundleDirectory, bundleId)},
+                    { "AzureFunctionsJobHost:extensionBundle:downloadPath", ExtensionBundleHelper.GetBundleDownloadPath(bundleId) },
```

Comment:
should this be double underscore instead of `:`

Reply:
no `:` is the delimiter used by dotnet runtime to specify hierarchy. Putting double underscore would break the bundle options logic as it will all be in the same level.